bin/pic-configure

```diff
@@ -165,6 +165,12 @@ if [ -f "$extension_param/cmakeFlags" ] ; then
         echo "       Is the file executable? (chmod u+x cmakeFlags)" >&2
         exit 2
     fi
+    # save the cmakeFlags setup number for future reference
+    if [ -f "$extension_param/cmakeFlagsSetup" ] ; then
+      rm $extension_param/cmakeFlagsSetup
+    fi
+    touch $extension_param/cmakeFlagsSetup
+    echo "Last configured cmakeFlags setup was: $cmakeFlagsNr" > $extension_param/cmakeFlagsSetup
```

Comment:
I would add additionally the content of `cmake_flags` too (on a new line), that `cmakeFlagsSetup` becomes more human-readable.

Reply:
Please add the content of the cmake_flags too, the number itself is not a good description.